src/context/intermediate/modals/Prompt.tsx

```diff
@@ -157,6 +158,7 @@ export const SpecialPromptModal = observer((props: SpecialProps) => {
                                         case "delete_channel":
```

Comment:
I don't think redirecting to home is the best idea for channel deletions - probably best to split this into its own category

Reply:
Honestly didn't even realise that delete channel was there lmao
I'll put a diff catch in there for that